lib/engine/stock_market.rb

```diff
@@ -40,14 +40,18 @@ def set_par(corporation, share_price)
       corporation.original_par_price = share_price
     end
 
+    def reroute_right_move(r, c)
```

Comment:
i don’t understand this method name, what does it mean?

Reply:
Given coordinates on the stock market it tells if we're at the rightmost spot, which means we should move up instead of right.

Reply:
def right_ledge?

Reply:
Thanks. Updated